### DISCUSSION TITLE
L2 Cache Error Injection Example

### DISCUSSION THREAD

**Discussion by bkzshabbaz** [polarfire-soc/polarfire-soc-documentation#47] [Bare metal embedded software] [answered]
Would it be possible to get example code for the L2 cache error injection feature?

**Comment by nitindeshpande** [comment, marked as answer]
Hi @bkzshabbaz, Presently we do not have a plan to publish any example on Cache error injection. 
If I may ask, could you provide some more detail about what you are trying to accomplish?
